tagbot/action/repo.py

```diff
@@ -179,8 +179,9 @@ def _registry_pr(self, version: str) -> Optional[PullRequest]:
             return None
         name = self._project("name")
         uuid = self._project("uuid")
+        sha = self.commit_sha_of_version(version)
         # This is the format used by Registrator/PkgDev.
-        head = f"registrator/{name.lower()}/{uuid[:8]}/{version}"
+        head = f"registrator-{name.lower()}-{uuid[:8]}-{version}-{sha}"
```

Comment:
Can you maybe add a link to the place in the RegistryTools.jl source code where this is defined?

I think it's https://github.com/JuliaRegistries/RegistryTools.jl/blob/0de7540015c6b2c0ff31229fc6bb29663c52e5c4/src/utils.jl#L23-L23

Reply:
Added